machine-learning/01-Intro to Machine Learning.ipynb

```diff
@@ -1,402 +1,392 @@
+        "1. **Deploy**  \n",
+        "  After you've trained a model ... it's just .NET Code! Build, Ship It - however you currently deploy your application."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## How do I get started?\n",
+        "Below we have a quick introduction to ML.NET - \"Hello ML.NET World!\" and the next three Notebooks in the series take a deep dive into [Data Prep and Feature Engineering](https://raw.githubusercontent.com/dotnet/csharp-notebooks/fa302c12c7494e5f8a5fdbe5d8283d8ff1fb7009/machine-learning/02-Data%20Preparation%20and%20Feature%20Engineering.ipynb), [Training and AutoML](https://raw.githubusercontent.com/JakeRadMSFT/csharp-notebooks/main/machine-learning/03-Training%20and%20AutoML.ipynb), and [Model Evaluation](https://raw.githubusercontent.com/dotnet/csharp-notebooks/fa302c12c7494e5f8a5fdbe5d8283d8ff1fb7009/machine-learning/04-Model%20Evaluation.ipynb)  "
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Hello ML.NET World!\n",
+        "The code in the following snippet demonstrates the simplest ML.NET application. This example constructs a linear regression model to predict house prices using house size and price data. \n",
+        "\n",
+        "First step is to reference the [Microsoft.ML](https://www.nuget.org/packages/Microsoft.ML/) package.\n",
+        "\n",
+        "Regarding this notebook, we add the reference to the package reference as follow:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "#r \"nuget: Microsoft.ML, 1.7.1\""
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "The second step is to reference the ML.NET namespaces:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "using Microsoft.ML;\n",
+        "using Microsoft.ML.Data;"
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Now we are ready to write the code to achieve the machine learning task we need to do. Always start with creating the [MLContext](https://docs.microsoft.com/dotnet/api/microsoft.ml.mlcontext?ranMID=43674&ranEAID=rl2xnKiLcHs&ranSiteID=rl2xnKiLcHs-LuTsrQLVgyEOYaht34D47g&epi=rl2xnKiLcHs-LuTsrQLVgyEOYaht34D47g&irgwc=1&OCID=AID2200057_aff_7795_1243925&tduid=(ir__2m3q0nl02wkf6gcatnnkkvci0e2xvxwafx3xgf9200)(7795)(1243925)(rl2xnKiLcHs-LuTsrQLVgyEOYaht34D47g)()&irclickid=_2m3q0nl02wkf6gcatnnkkvci0e2xvxwafx3xgf9200&view=ml-dotnet) which is the common context for all ML.NET operations"

```

Comment:
another weird link

Reply:
I'll fix these with ntbk.io short urls once we've settled on content a bit :).